sklearn/feature_selection/univariate_selection.py

```diff
@@ -596,7 +596,7 @@ def _get_support_mask(self):
         n_features = len(self.pvalues_)
         sv = np.sort(self.pvalues_)
         selected = sv[sv <= float(self.alpha) / n_features
-                      * np.arange(n_features)]
+                      * (np.arange(n_features) + 1)]
```

Comment:
I'd be happy to disable W503. I tend to agree with the more recent convention in terms of readability.

Strange that https://www.python.org/dev/peps/pep-0008 doesn't mention when the document was updated.